client/src/app/shared/pipes/parse-poll-number.pipe.ts

```diff
@@ -18,6 +18,7 @@ export class ParsePollNumberPipe implements PipeTransform {
     public constructor(private translate: TranslateService) {}
 
     public transform(value: number): number | string {
+        value = Number(value);
```

Comment:
What type was `value`? And why does it have the wrong type?

Reply:
was string, hat a format like: 1.000000
if coming through the detail component and using the real model, value was a number.
if coming through the projector, value was a string.

Both were using the `os-motion-poll-detail-content`

Reply:
Ah ok, I didn't know that it is a decimal field. The models explicitly convert decimal fields to float: https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/shared/models/base/base-decimal-model.ts#L10

Maybe you should use `parseFloat` to have a uniform conversion method. The IMO better fix would be in the projector slides: Since there it is a number in the decimal-string-format, it should be converted there. The interface `BasePollSlideData` says that `votesvalid`, ... are numbers, which is wrong for the raw projector data. But if it were converted, it is ok.

The conversion can easily be done in the `BasePollSlideComponent`. In `public set data`, you can do the same as in the `BaseDecimalModel` class: Iterate over a fixed array of fields (like `votecast`, ...) and parse them. E.g. for the assignment poll model, these fields  are specified here: https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/shared/models/assignments/assignment-poll.ts#L32

Reply:
The solution I provided here for the pipe is adequate to resolve the given problem.
Since you have a clear idea what to adjust and where you might just as well take over this PR.

To convert other datatypes to number in Typescript, `Number()` would be the way I recommend.
https://i.stack.imgur.com/QNMfI.png

Reply:
Ok, I'll add a commit.